src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/regional.yml

```diff
@@ -89,6 +89,19 @@ Resources:
             Condition:
               StringEquals:
                 aws:PrincipalOrgID: !Ref OrganizationId
+  TerraformLockTable:
```

Comment:
This should be optional too, only for customers using Terraform.

Reply:
Added a parameter in adfconfig.yml for Terraform extensions and a condition to deploy DynamoDB table only if the parameter is set to True